src/Components/SignUpPage/Register.js

```diff
@@ -12,7 +12,7 @@ const Register = ({ handleCloseModal }) => {
   });
 
   const handleChange = (e) => {
-    setUserDetails({ ...userDetails, [e.target.name]: e.target.value });
+    setUserDetails((userDetails) => ({ ...userDetails, [e.target.name]: e.target.value }));
```

Comment:
Change the name

```suggestion
    setUserDetails((prev) => ({ ...prev
```
OR
```suggestion
    setUserDetails((previous) => ({ ...previous
```
OR
```suggestion
    setUserDetails((previousState) => ({ ...previousState
```

Reply:
okay sir